### PR TITLE
Glimmer support

### PR DESCRIPTION
**Pull request by jasonmit**
/cc @al3x-edge

* [x] fix initializer to fallback back to `Ember.HTMLBars._registerHelper` when `Ember.Helper` is `undefined`
* [ ] make tests backwards compat (ember-try) currently only passing in 1.13 now

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: app/initializers/ember-intl.js [hunk 1]
```diff
@@ -23,12 +23,12 @@ export function initializer(registry, app) {
         registry.register('adapter:-intl-adapter', IntlAdapter);
     }
 
-    Ember.HTMLBars._registerHelper('format-date', FormatDate);
-    Ember.HTMLBars._registerHelper('format-time', FormatTime);
-    Ember.HTMLBars._registerHelper('format-relative', FormatRelative);
-    Ember.HTMLBars._registerHelper('format-number', FormatNumber);
-    Ember.HTMLBars._registerHelper('format-html-message', FormatHtmlMessage);
-    Ember.HTMLBars._registerHelper('format-message', FormatMessage);
+    registry.register('helper:format-date', FormatDate);
```
Comment (jasonmit) [inline app/initializers/ember-intl.js]: This will likely not be backwards compat

File: addon/legacy/helpers/intl-get.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import {
+    Stream,
+    read,
+    readHash,
+    destroyStream
+} from 'ember-intl/utils/streams';
+
+// Backwards compatibility for Ember < 1.13
+export default function (value, options) {
```
Comment (jasonmit) [inline addon/legacy/helpers/intl-get.js]: I really hate this method.. maybe I'll attempt cleaning it up once i get the tests passing

File: addon/helpers/base.js [hunk 3]
```diff
@@ -4,73 +4,48 @@
...
+                let intl = this.get('intl');
+
+                if (hash && hash.format) {
+                    format = intl.getFormat(formatType, hash.format);
+                }
+
+                return this.formatter.format.call(
+                    this.formatter,
+                    value,
+                    Ember.$.extend(Ember.getProperties(intl, 'locale'), format, hash)
```
Comment (jasonmit) [inline addon/helpers/base.js]: I want to remove use of jQuery for extend

**Comment by al3x-edge** [comment]
Awesome stuff @jasonmit =] :+1:

**Comment by jasonmit** [comment]
And we are green :smile:

**Comment by al3x-edge** [comment]
:dancers: Awesome! Thanks for the hard work. Felt a double buzz in my pocket -- had to check.

**Pull request merged**